src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -0,0 +1,335 @@
+                                            </span>
+                                    ))}
+                                </span>
+                            )}
+                        </span>
+                    )}
+                    </div>
+                ))}
+                { clinicalGroupMatch.notMatches.length > 0 &&
+                    <DefaultTooltip
+                        placement='bottomLeft'
+                        trigger={['hover', 'focus']}
+                        overlay={this.tooltipGenomicContent(clinicalGroupMatch.notMatches)}
+                        destroyTooltipOnHide={false}
+                        onPopupAlign={placeArrowBottomLeft}
+                    >
+                        {this.mainContent('No alterations in ' + this.getHugoSymbolName(clinicalGroupMatch.notMatches, 3) + ' defined by the trial')}
+                    </DefaultTooltip>
+                }
+            </span>
+        );
+    }
+
+    public tooltipGenomicContent(data: Array<IGenomicGroupMatch>) {
+        const props = this.props;
+        return (
+            <div style={{ maxHeight:400, maxWidth:600, overflow:'auto' }}>
+                {data.map((genomicGroupMatch: IGenomicGroupMatch) => (
+                    <div className={styles.genomicSpan}><b>Not </b>{genomicGroupMatch.genomicAlteration.replace(/!/g, '') + ' '}
+                        {genomicGroupMatch.matches[0].sampleIds.map((sampleId: string) => (
+                            <span className={styles.genomicSpan}>
+                                {props.sampleManager!.getComponentForSample(sampleId, 1, '')}
+                            </span>
+                        ))}
+                    </div>
+                ))}
+            </div>
+        );
+    }
+    public tooltipClinicalContent(data: Array<string>) {
+        return (
+            <div style={{ maxHeight:400, maxWidth:600, overflow:'auto' }}>
+                <ul className={styles.alterationUl}>
+                    {data.map((cancerType: string) => (
+                        <li>{cancerType}</li>
+                    ))}
+                </ul>
+            </div>
+        );
+    }
+
+    public mainContent(content: string) {
+        return (
+            <span>
+                <a>{content}</a>
+            </span>
+        );
+    }
+
+    public getHugoSymbolName(matches: Array<IGenomicGroupMatch>, threshold: number) {
+        const hugoSymbolSet = new Set([...matches].map(x => x.genomicAlteration.split(' ')[0].slice(1)));
+        if (hugoSymbolSet.size <= threshold) {
+            return [...hugoSymbolSet].join(', ');
+        }
+        return hugoSymbolSet.size + ' genes' ;
+    }
+
+    public buildDetailedTrialMatches() {
+        const props = this.props;
+        const groupByTrialId = _.groupBy(props.trialMatches, 'nctId');
+        let matchedTrials: Array<IDetailedTrialMatch> = [];
```

Comment:
This matchedTrials should be tested. Some unit tests need to be added  to test priority and other